net_io.c

```diff
@@ -390,7 +390,7 @@ static void modesSendBeastOutput(struct modesMessage *mm) {
 
 static void send_beast_heartbeat(struct net_service *service)
 {
-    static char heartbeat_message[] = { 0x1a, '1', 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+    const char heartbeat_message[] = { 0x1A, '1', 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
This one doesn't seem like a great idea, now that I'm looking at disassembly. It was mainly for clarity- one could mark these `static const` to better convey intent, but it isn't really necessary. I'll drop this commit.